CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+## How to contribute code
+
+Follow these steps to submit your code contribution.
+
+### Step 1. Open an issue
+
+Before making any changes, we recommend opening an issue (if one doesn't already
+exist) and discussing your proposed changes. This way, we can give you feedback
+and validate the proposed changes.
+
+If your code change involves the fixing of a bug, please include a 
+[Colab](https://colab.research.google.com/) notebook that shows
+how to reproduce the broken behavior.
+
+If the changes are minor (simple bug fix or documentation fix), then feel free
+to open a PR without discussion.
+
+### Step 2. Make code changes
+
+To make code changes, you need to fork the repository. You will need to setup a
+development environment and run the unit tests. This is covered in section
+"Setup environment".
+
+### Step 3. Create a pull request
+
+Once the change is ready, open a pull request from your branch in your fork to
+the master branch in [keras-team/keras-cv](https://github.com/keras-team/keras-cv).
+
+### Step 4. Sign the Contributor License Agreement
+
+After creating the pull request, the `google-cla` bot will comment on your pull
```

Comment:
"the `google-cla` bot will comment" -> unfortunately this is not true at this time. Do you know more about why it has been disabled? In any case, here it would be helpful to add a link to the CLA website that users can follow if they don't get a prompt on the PR (which is the current situation). Here: https://cla.developers.google.com/clas

Reply:
Interesting, I didn't realize that the bot was disabled until now.  That's no good.  I added a comment linking to the CLA website.

Also, note that this is currently the in the keras-team/keras CONTRIBUTING.md guide as well @fchollet.  Maybe we need to update this as well.